src/mantle/framework/console/class-generator-command.php

```diff
@@ -214,7 +221,7 @@ protected function get_folder_path( string $name ): string {
 			$parts = '';
 		}
 
-		return \untrailingslashit( $this->get_base_path() . strtolower( str_replace( '\\', '/', $this->type ) ) . '/' . $parts );
+		return \untrailingslashit( untrailingslashit( $this->get_base_path() ) . '/' . strtolower( str_replace( '\\', '/', $this->type ) ) . '/' . $parts );
```

Comment:
I don't know if this is any cleaner...
```suggestion
		return \untrailingslashit( trailingslashit( $this->get_base_path() ) . strtolower( str_replace( '\\', '/', $this->type ) ) . '/' . $parts );
```